src/main/java/org/mockbukkit/mockbukkit/entity/PlayerMock.java

```diff
@@ -661,6 +663,24 @@ public EntityEquipment getEquipment()
 		return (EntityEquipment) getInventory();
 	}
 
+	@Override
+	public boolean hasCooldown(@NotNull ItemStack itemStack)
+	{
+		return false;
```

Comment:
Implementation of non parity behavior, if possible throw an unimplemented operation exception here